Fixed warning of line 300 in Page.ts

tasneemkoushar opened the pull request:
On running "npm run generate-docs",  getting warning "The TSDoc tag "@see" is not supported by this tool" in line 300 on "Page.ts"
As @see is not supported by TSDoc so I changed it to {@link}

tasneemkoushar committed:
Fixed warning of line 300 in Page.ts

On running "npm run generate-docs",  getting warning "The TSDoc tag "@see" is not supported by this tool" in line 300 on "Page.ts"
As @see is not supported by TSDoc so I changed it to {@link}

tasneemkoushar committed:
Fixed warning on line 37 of LifecycleWatcher.ts

There was a warning regarding "ProtocolLifeCycleEvent" of not being exported in LifecycleWatcher.ts.
I fixed that and also gave the TSDoc tag of being @internal to it.

tasneemkoushar committed:
Fixed warning in 27 of "Browser.ts"

Exported "BrowserCloseCallback" to fix one of the warnings in Browser.ts.

tasneemkoushar committed:
Update Page.ts

tasneemkoushar committed:
Update LifecycleWatcher.ts

tasneemkoushar committed:
Removed @see tag from line 300 in "Page.ts"

jackfranklin commented:
@tasneemkoushar thanks for this PR ! I think the issue is that your git commit messages aren't passing the check:

```
> puppeteer@8.0.0-post commitlint /home/runner/work/puppeteer/puppeteer
> commitlint --from=HEAD~1

⧗   input: Removed @see tag from line 300 in "Page.ts"
✖   subject may not be empty [subject-empty]
✖   type may not be empty [type-empty]

✖   found 2 problems, 0 warnings
ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint
```

If you change the last commit message to be something like:

```
fix: remove invalid @see documentation reference
```

Then I think commitlint should be happy.

tasneemkoushar commented:
@jackfranklin Sir you have approved my changes, but I can see some checks were not successful, most probably because of .eslintrc
code style. Will my code gets merged with the present state or do I need to fix these checks?

tasneemkoushar committed:
fix(Errors.ts):  exported class CustomError on line 20

CustomError was not exported from Errors.ts, due to which we were warning "ae-missing-release-tag".

tasneemkoushar committed:
docs: exported class CustomError from Errors.ts

tasneemkoushar committed:
Add files via upload

tasneemkoushar committed:
Delete puppeteer.accessibility.md

tasneemkoushar committed:
fix: exported class MediaFeature from page.ts

tasneemkoushar committed:
fix: exported class MediaFeature from page.ts

tasneemkoushar committed:
fix: exported CDPSessionOnMessageObject from Connection.ts

tasneemkoushar committed:
fix: exported Device from DeviceDescriptors.ts

tasneemkoushar committed:
fix: exported ConsoleAPICalledCallback and ExceptionThrownCallback from WebWorker.ts

jackfranklin commented:
@tasneemkoushar I think we can close this PR now as you've made more since that have solved the issues :)
